hera_mc/cm_hookup.py

```diff
@@ -363,7 +369,8 @@ def is_fully_connected(self, hookup_dict, any_or_all='all'):
         else:
             return num_fully_connected > 0
 
-    def show_hookup(self, hookup_dict, cols_to_show='all', show_levels=False, show_ports=True, show_revs=True):
+    def show_hookup(self, hookup_dict, cols_to_show='all', show_levels=False, show_ports=True, show_revs=True,
+                    write_to_file=False):
```

Comment:
Instead of a filename flag, the new argument should be called `file` and optionally receive a file-like object (e.g. the return value from the `open()` function) as an argument. Its default value should be `None`, in which case this function should override the argument's value with `sys.stdout`. Then, the `print` calls should use the `file` keyword argument to specify where the printed text should go:

```
print('foo', file=file)
```

Finally, there should be a separate `output_format` keyword that defaults to `"ascii"` but can take on the value `"html"`, in which case the outputted text should have the wrapping HTML tags added.